stglib/eofe.py

```diff
@@ -533,12 +536,27 @@ def calc_cor_brange(ds):
     # Correct using adjusted sound speed
     # distance (m) = time (sec) x sound speed (m/sec)
     time_sec = xr.DataArray((ds.Altitude_m) / ds.attrs["SoundSpeed_mps"])
```

Comment:
Is "SoundSpeed_mps" something that comes from the instrument, or is it input by the user in a config file?

Reply:
SoundSpeed_mps comes from the instrument log file.